nightfall-client/src/services/finalise-withdrawal.mjs

```diff
@@ -36,19 +36,15 @@ export async function finaliseWithdrawal(transactionHash) {
   );
 
   const shieldContractInstance = await getContractInstance(SHIELD_CONTRACT_NAME);
-  try {
-    const rawTransaction = await shieldContractInstance.methods
-      .finaliseWithdrawal(
-        buildSolidityStruct(block),
-        Transaction.buildSolidityStruct(transactions[index]),
-        index,
-        siblingPath,
-      )
-      .encodeABI();
+  const rawTransaction = await shieldContractInstance.methods
```

Comment:
It seems like we miss the try-catch block here?

Reply:
Yes @signorecello , a `try...catch` should do something, otherwise, it is useless. Look that it did just thrown the error again, doing nothing.

Reply:
You're right, the newest node version will also stop the process on an unhandled exception which makes it essencially the same thing